content/en/sensitive_data_scanner/guide/investigate_sensitive_data_issues.md

```diff
@@ -25,42 +25,79 @@ Sensitive Data Scanner is a stream-based, pattern matching service used to ident
 If you want to modify the Scanning Rule that was used to detect the sensitive data issue, click **Modify Rule** at the top of the panel.
 
 Additionally, you can also:
-- Use [Case Management][2] to track, triage, and investigate the issue, click **Create Case** at the top of the panel. Associated cases are surfaced in the Summary page.
-- Use [Incident Management][3] to create an incident, you can add the issue to an existing incident or declare a new incident. Click the **Declare Incident** dropdown menu to add the issue to an existing incident. Click **Declare Incident** to declare a new incident.
-- Use [Audit Trail][4] to see who may have accessed this sensitive data within Datadog, **View in Audit Trail** in the **Users who accessed these events** section.
+- Use [Case Management][1] to track, triage, and investigate the issue, click **Create Case** at the top of the panel. Associated cases are surfaced in the Summary page.
+- Use [Incident Management][2] to create an incident, you can add the issue to an existing incident or declare a new incident. Click the **Declare Incident** dropdown menu to add the issue to an existing incident. Click **Declare Incident** to declare a new incident.
+- Use [Audit Trail][3] to see who may have accessed this sensitive data within Datadog, **View in Audit Trail** in the **Users who accessed these events** section.
 
 {{< img src="sensitive_data_scanner/investigate_sensitive_data_issues/case_mgmt_02_01_2024.png" alt="The case page showing information about the security issue, the assignee and creator of the case, and a timeline of events" style="width:60%;">}}
 
+[1]: /service_management/case_management/
+[2]: /service_management/incident_management/
+[3]: /account_management/audit_trail
+
+{{% /tab %}}
+{{% tab "Cloud Storage" %}}
+
+Click the **Datastores with Sensitive Data** tab to see all sensitive data issues for Cloud Storage.
+
+In the **xxx Datastores with Sensitive section**, click on any of the dropdown menus to filter on datastores based on the type of sensitive data, account, region, team and so on.
+
+To investigate a datastore:
+
+1. Click on a datastore.
+1. You can view files where sensitive data was found and then click on a file to inspect it in AWS.
+  Datadog recommends doing the following:
+    - Review a few files to get a sense of the classification accuracy.
+    - Follow up with the team or service owner (shown in the side panel) to confirm if sensitive data is supposed to be in that bucket.
+      - If it is not supposed to be in that bucket, delete the files or move them to an appropriate bucket.
+      - If it is supposed to be in that bucket, follow the following steps to improve your security posture:
+
+        1. Click the **Security** tab in the side panel and review the **Misconfigurations** section.
+        1. Click on a misconfiguration to see details in Cloud Security Management.
+        1. In the **Next Steps** section:
```

Comment:
I think the steps that follow this one may not be indented correctly.

Reply:
Thanks! Fixed it.